File: node/domain/user/PhoneNumber.ts

const phoneNumberRegex = new RegExp("[0-9]\g")
export default class PhoneNumber{
    constructor(public value){
        if(!(value instanceof String)){
            throw new Error("Phone Number must be a string")
        }
        if(!value.match(phoneNumberRegex)){
            throw new Error("Phone Number must be included by digits")
        }
        if(value.length != 10){
            throw new Error("Phone Number length must be 10")
        }
    }
}

File: python/domain/value_objects/phone_number.py
class PhoneNumber:
    def __init__(self, value):
        if not isinstance(value, str):
            raise ValueError("Phone Number must be a string")
        if not re.match(r"^\d{10}$", value):\
            raise ValueError("Phone number must be exactly 10 digits")
        self._value = value
        